server/protocols/jmap/src/main/java/org/apache/james/jmap/model/CreationMessage.java

```diff
@@ -363,6 +364,11 @@ private void assertValidFromProvided(ImmutableList.Builder<ValidationResult> err
     public boolean isIn(MessageManager mailbox) {
         return mailboxIds.contains(mailbox.getId().serialize());
     }
+
+    public boolean isInOnly(MessageManager mailbox) {
```

Comment:
isOnlyIn ?

Reply:
Any proposal for a better name?

Reply:
`isTheOnlyOne` or `highlander`

Reply:
you don't like mine ?

Reply:
Oh crap I did not understood your proposal. Looks good to me.